docs/_user_guide/best_practices_migrating_from_jupyter.md

```diff
@@ -79,37 +85,37 @@ In JupyterHub, you should see the new kernel, `my_env`, available.
 
 Try running your JupyterHub notebook using the `my_env` kernel you just installed.  You can change the kernel of an existing notebook by going to Kernel->Change Kernel...
 
-<img src="/images/jupyterhub_changekernel.png" alt="Change Kernel">
+![Change Kernel](/assets/images/jupyterhub_changekernel.png)
 
 If it works on this kernel, your next step would be to migrate these commands to a `.py` script.  You can test this by activating your `venv` environment on the Yens, and running your script via `python <my_script.py>`.
 
 #### Write a submit script to run your process on `yen-slurm` using your working environment
 
-The specifics of writing a submit script are <a href="/yen/scheduler.html">outlined here</a>.  In addition, you'll need to make sure your submit script is running the correct python environment.  There are two ways to do that.
+The specifics of writing a submit script are [`outlined here`](/_user_guide/slurm/){target="_blank"}.  In addition, you'll need to make sure your submit script is running the correct python environment.  There are two ways to do that.
 
 First, you can run 
-```bash
-$ source <path/to/your/venv>/bin/activate
+```title="Terminal Command"
+source <path/to/your/venv>/bin/activate
 ``` 
 before you run `python`.  Afterwards, you can add the line 
-```bash
-$ echo $(which python);
+```title="Terminal Command"
+echo $(which python);
 ``` 
 to print out which python your script using, to be sure it's in `my_env/bin/`.
 
 
-The second method is to explicitly call out the python instance you want to use.  In your submit script, instead of using the command 
-```bash
-$ python <my_script.py>
+The second method is to explicitly call out the python instance you want to use.  In your submit script, instead of using the command
+```{ .yaml .no-copy }
+python <my_script.py>
 ```
-, you would use 
-```bash
-$ <path/to/your/venv>/bin/python <my_script.py>
+you would use 
+```{ .yaml .no-copy }
+<path/to/your/venv>/bin/python <my_script.py>
 ``` 
 to be sure the `python` instance in your `venv` is being used.
 
 
-## Non-Python Code Migration
+### Non-Python Code Migration
```

Comment:
I don't think this is a useful section..

Reply:
deleted